src/main/java/info/movietrash/cinemabase/converter/impl/UserMovieConverterImpl.java

```diff
@@ -0,0 +1,51 @@
+package info.movietrash.cinemabase.converter.impl;
+
+import info.movietrash.cinemabase.converter.UserMovieConverter;
+import info.movietrash.cinemabase.dto.MovieDto;
+import info.movietrash.cinemabase.model.Genre;
+import info.movietrash.cinemabase.model.Movie;
+import info.movietrash.cinemabase.model.User;
+import info.movietrash.cinemabase.model.UserMovie;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class UserMovieConverterImpl implements UserMovieConverter {
+
+    @Override
+    public UserMovie createUserMovie(User user, Movie movie, MovieDto movieDto) {
+        if ((user == null) || (movie == null) || (movieDto == null)) {
```

Comment:
allNotNull(Object... values) is better

Reply:
if (ObjectUtils.allNotNull(user, movie, movieDto))---- done